src/ChannelServer/Database/Account.cs

```diff
@@ -54,6 +60,33 @@ public Account()
 		{
 			this.Settings = new AccountSettings();
 			this.Variables = new Variables();
+			this._chatMacros = new List<ChatMacro>();
```

Comment:
`this` should only be used for publics, not privates that are already prefixed with an underscore. Though I recently thought that it might make sense to actually use `this` for them as well, *instead* of prefixing them, for consistency, and because I see this mistake made all the time with my chosen code conventions^^

Reply:
I'm fine with using it only with `publics`; it is not a problem :)